docs/platform/howto/use-google-private-service-connect.rst

```diff
@@ -0,0 +1,175 @@
+
+PSC_CONNECTION_ID is the identifier assigned to Google for the connection, and you can use it to verify that the connection is indeed matching your PSC endpoint.
+
+PRIVATELINK_CONNECTION_ID is Aiven internal identifier for the connection, you'll need in the final connection approval step below.
+
+To enable a connection, you should approve it, at the same time providing the IP address assigned to your PSC endpoint - whether automatically assigned or static. Aiven will use this IP address for pointing the service DNS records necessary for the clients to reach the Aiven service through the PSC connection.
+
+To complete the connection, you need to issue approval command as follows:
+
+.. code:: shell
+
+    avn privatelink google connection approve MY_SERVICE_NAME --privatelink-connection-id PRIVATELINK_CONNECTION_ID --user-ip-address PSC_ENDPOINT_IP_ADDRESS
+
+The connection will initially transition to user-approved state:
+
+.. code:: shell
+
+    avn service privatelink google connection list MY_SERVICE_NAME
+
+.. code:: shell
+
+    PRIVATELINK_CONNECTION_ID  PSC_CONNECTION_ID  STATE          USER_IP_ADDRESS
+    =========================  =================  =============  ===============
+    plc3fd852bec98             12870921937223780  user-approved  10.0.0.100
+
+An extra
+
+.. code:: shell
+
+    avn service privatelink google refresh MY_SERVICE_NAME
+
+may be needed at this point, because updates to service attachment accept lists are not immediately reflected in the states of returned connected endpoints.
+
+After connection has been established and DNS records have been populated, connection will appear as 'active':
+
+.. code:: shell
+
+    avn service privatelink google connection list MY_SERVICE_NAME
+
+.. code:: shell
+
+    PRIVATELINK_CONNECTION_ID  PSC_CONNECTION_ID  STATE   USER_IP_ADDRESS
+    =========================  =================  ======  ===============
+    plc3fd852bec98             12870921937223780  active  10.0.0.100
+
+The state of your Private Service Connect endpoint should have transitioned from 'pending' to 'accepted' at this point.
+
+PSC connectivity has now been established. As the final step, you'll need to allow connectivity using the PSC endpoint.
+
+Step 4: Enable Private Link access service components
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Finally, enable Private Link access on your Aiven services using either the Aiven CLI or `Aiven Console <https://console.aiven.io/>`_.
+
+**Aiven CLI**
+
+To enable Private Service Connect access for your service in the Aiven CLI, set ``user_config.privatelink_access.<service component>`` to true for the components you want to enable. For example, for Apache Kafka the command is:
+
+.. code:: shell
+
+    avn service update -c privatelink_access.kafka=true MY_SERVICE_NAME
+
+**Aiven Console**
+
+To enable Private Link access in the `Aiven Console <https://console.aiven.io/>`_:
```

Comment:
To enable Private Link access in the `Aiven Console <https://console.aiven.io/>`_, take the following steps: